sormas-api/src/main/resources/captions.properties

```diff
@@ -2233,11 +2233,11 @@ sormasToSormasSendLabMessage=Send to another organization
 BAGExport=BAG Export
 
 # Survnet Gateway
-SurvnetGateway.title=SurvNet
-SurvnetGateway.send=send to own SurvNet instance
-SurvnetGateway.unableToSend=Unable to send
-SurvnetGateway.confirmSend=Confirm sending
-SurvnetGateway.sendShort=Send to SurvNet
+ExternalSurveillanceToolGateway.title=Reporting Software
```

Comment:
I think we should rename it to either `Reporting System` or `Reporting Tool` because this wording it's too long for the UI.
![image](https://user-images.githubusercontent.com/3462897/111614247-091d2700-87e8-11eb-905c-e0d580130897.png)
Ideally would be to adjust the UI for any wording length, but I would be ok with just the renaming also.